addons/project_mrp/models/project_project.py

```diff
@@ -59,23 +59,22 @@ def action_view_mrp_production(self):
     def _get_stat_buttons(self):
         buttons = super()._get_stat_buttons()
         if self.env.user.has_group('mrp.group_mrp_user'):
-            self_sudo = self.sudo()
             buttons.extend([{
                 'icon': 'flask',
                 'text': self.env._('Bills of Materials'),
-                'number': self_sudo.bom_count,
+                'number': self.bom_count,
```

Comment:
no longer need `sudo`? 🧐

Reply:
It's the same idea, if we keep the `sudo()` here, it will bypass the multi-company `ir.rule`
To me it's okay to get rid of it because we check the that the user has the `mrp.group_mrp_user` group